Library/Formula/smpeg2.rb

```diff
@@ -0,0 +1,46 @@
+class Smpeg2 < Formula
+  desc "SDL MPEG Player Library"
+  homepage "http://icculus.org/smpeg/"
+  url "svn://svn.icculus.org/smpeg/tags/release_2_0_0/"
+
+  head "svn://svn.icculus.org/smpeg/trunk"
+
+  option "with-plaympeg", "Build plaympeg application"
+  if build.with? "plaympeg"
+    conflicts_with "homebrew/head-only/smpeg", :because => "both install `plaympeg` binaries."
```

Comment:
smpeg used to be in the core and there is a pull request to get it back: Homebrew/homebrew-head-only#79. Also I didn't want it to interfere with existing smpeg installations, but if it was overconcerned for `head-only`, I'd not be against dropping `conflicts_with`.